plugins/Tools/RotateTool/RotateTool.py

```diff
@@ -264,6 +268,23 @@ def setRotationSnapAngle(self, angle):
             self._snap_angle = angle
             self.propertyChanged.emit()
 
+    ##  Wether the rotate tool is in 'Lay flat by face'-Mode.
+    #
+    #   \return (bool)
+    def getSelectFaceToLayFlatMode(self):
```

Comment:
Missing mypy typing.

Reply:
Then we should do the typing of all the tools :-)

Reply:
Oh, I figured this would fall under the boy scouting rule.

Reply:
Yes, you're right! Ok, will do. It just always looks so lopsided to me when _the entire rest of the file_ is not typed :'-D

Reply:
Well, you could also do the rest ofcourse (and be a _really_ good boyscout)

Reply:
... the entire class proved a bit much...